klippa/src/parsing_util.rs

```diff
@@ -132,3 +175,29 @@ fn test_parse_drop_tables() {
     let output = parse_drop_tables(input).unwrap();
     assert!(output.is_empty());
 }
+
```

Comment:
nit: you appear to support (intentionally? - it's a weird input, should it work) but not test mixed delimiters, e.g. "7,8 9"